StripeFinancialConnections/StripeFinancialConnections/Source/Web/FinancialConnectionsWebFlowViewController.swift

```diff
@@ -98,9 +103,9 @@ final class FinancialConnectionsWebFlowViewController : UIViewController {
         continueStateView.isHidden = true
         view.addSubview(continueStateView)
         NSLayoutConstraint.activate([
-            continueStateView.leadingAnchor.constraint(equalTo: view.safeAreaLayoutGuide.leadingAnchor, constant: 24),
-            continueStateView.trailingAnchor.constraint(equalTo: view.safeAreaLayoutGuide.trailingAnchor, constant: -24),
-            continueStateView.bottomAnchor.constraint(equalTo: view.safeAreaLayoutGuide.bottomAnchor, constant: -20),
+            continueStateView.leadingAnchor.constraint(equalTo: view.safeAreaLayoutGuide.leadingAnchor, constant: 0),
```

Comment:
not sure if you like using the helper functions, but if you are ok with them, maybe all of this code can be replaced with :

`view.addAndPinSubviewToSafeArea(continueStateView)`

Reply:
ah yeah, I hate these functions just copied that over. Will use that instead.